pages/index.js

```diff
@@ -25,11 +28,71 @@ export default function HomePage({
         setRandomSurprise={setRandomSurprise}
         handleSurprise={handleSurprise}
       />
-      {randomSurprise ? (
-        <SpotlightCard randomSurprise={randomSurprise} />
-      ) : null}
-      {formResults.length > 0 && <PreviewCard formResults={formResults} />}
-      {formResults === -1 && <h2>no matches...</h2>}
-    </div>
+      {formResults === -1 && (
+        <StyledSectionEmpty>
+          <h2>no matches...</h2>
+        </StyledSectionEmpty>
+      )}
+      {!formResults && (
+        <StyledSectionEmpty>
+          <h2>Howdy, and welcome to our site!</h2>
+          <p>
+            At the moment you can search for a region, or an activity. Or both
+            if you like. If you find a bug, or want to give feedback, please
+            leave a comment at github.
+            <br /> Thanks, the Travel App Team Jeanny, Jeanette and Marcel.
+          </p>
+        </StyledSectionEmpty>
+      )}
+      {formResults.length > 0 && (
+        <StyledAvailableTrips>Available trips:</StyledAvailableTrips>
+      )}
+      <StyledSection>
+        {randomSurprise ? (
+          <SpotlightCard
+            randomSurprise={randomSurprise}
+            favoritePlaces={favoritePlaces}
+            onToggleFavorite={onToggleFavorite}
+          />
+        ) : null}
```

Comment:
Using the logical and would be another possibility: 

```suggestion
        {randomSurprise && (
          <SpotlightCard
            randomSurprise={randomSurprise}
            favoritePlaces={favoritePlaces}
            onToggleFavorite={onToggleFavorite}
          />
        )}
```